UnityProject/Assets/Scripts/Managers/LavaLandManager.cs

```diff
@@ -84,6 +84,8 @@ public IEnumerator SpawnLavaLandCo()
 	{
 		foreach (var script in randomGenScripts)
 		{
+			if (script == null) continue;
+
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
LavaLandManager+<SpawnLavaLandCo>d__17.MoveNext () (at /github/workspace/UnityProject/Assets/Scripts/Managers/LavaLandManager.cs:87)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at /home/builduser/buildslave/unity/build/Runtime/Export/Scripting/Coroutines.cs:17)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
LavaLandManager:SpawnLavaLand() (at /github/workspace/UnityProject/Assets/Scripts/Managers/LavaLandManager.cs:80)
EventManager:Broadcast(EVENT) (at /github/workspace/UnityProject/Assets/Scripts/Managers/EventManager.cs:94)
GameManager:StartRound() (at /github/workspace/UnityProject/Assets/Scripts/Managers/GameManager.cs:414)
AdminCommands.AdminCommandsManager:CmdStartRound(String, String) (at /github/workspace/UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs:98)
System.Reflection.MethodBase:Invoke(Object, Object[])
AdminCommands.ServerCommandVersionOneMessageClient:Process() (at /github/workspace/UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs:360)
GameMessageBase:Process(NetworkConnection) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
ClientMessage:Process(NetworkConnection) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Client/ClientMessage.cs:14)
GameMessageBase:PreProcess(NetworkConnection, GameMessageBase) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.<>c__DisplayClass6_0`2:<MessageHandler>b__0(NetworkConnection, NetworkReader, Int32) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection:InvokeHandler(Int32, NetworkReader, Int32) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection:TransportReceive(ArraySegment`1, Int32) (at /github/w